index.html

```diff
@@ -592,24 +607,34 @@ <h2><dfn>PerformanceObserverEntryList</dfn> interface</h2>
         <section>
           <h2><dfn>getEntries()</dfn> method</h2>
           <p>Returns a <a>PerformanceEntryList</a> object returned by <a>filter
-          buffer by name and type</a> algorithm with <a>this</a>'s <a>entry list</a>,
-          <var>name</var> and <var>type</var> set to <code>null</code>.</p>
+          buffer</a> algorithm with <a>this</a>'s <a>entry list</a>,
+          <var>name</var>, <var>type</var> and <var>navigationId</var> set to
+          <code>null</code>.</p>
         </section>
         <section>
           <h2><dfn>getEntriesByType()</dfn> method</h2>
           <p>Returns a <a>PerformanceEntryList</a> object returned by <a>filter
-          buffer by name and type</a> algorithm with <a>this</a>'s <a>entry list</a>,
-          <var>name</var> set to <code>null</code>, and <var>type</var> set to the
-          method's input <code>type</code> parameter.</p>
+          buffer</a> algorithm with <a>this</a>'s <a>entry list</a>,
+          <var>name</var> set to <code>null</code>, <var>type</var> set to the
+          method's input <code>type</code> parameter, and <var>navigationId</var>
+          set to <code>null</code>.</p>
         </section>
         <section>
           <h2><dfn>getEntriesByName()</dfn> method</h2>
           <p>Returns a <a>PerformanceEntryList</a> object returned by <a>filter
-          buffer by name and type</a> algorithm with <a>this</a>'s <a>entry list</a>,
+          buffer</a> algorithm with <a>this</a>'s <a>entry list</a>,
           <var>name</var> set to the method input <code>name</code> parameter, and
           <var>type</var> set to <code>null</code> if optional `entryType` is omitted,
           or set to the method's input <code>type</code> parameter otherwise.</p>
         </section>
+        <section>
+          <h2><dfn>getEntriesByNavigationId()</dfn> method</h2>
```

Comment:
Did we discuss creation of a dedicated filter, compared to applying a JS filter on `getEntries()`? I'm guessing this would have performance benefits, but it's worthwhile to articulate the reasoning (maybe in the explainer)